cpr/session.cpp

```diff
@@ -47,13 +48,15 @@ class Session::Impl {
     Url url_;
     Parameters parameters_;
     Proxies proxies_;
+    struct curl_httppost* formpost_;
```

Comment:
You are correct though that adding the destructor for the sake of deleting that raw pointer would remove move semantics, which `Session` currently has.